src/main/java/mate.academy/Main.java

```diff
@@ -10,7 +10,7 @@ public class Main {
     public static void main(String[] args) {
         // Please test your Injector here. Feel free to push this class as a part of your solution
         Injector injector = Injector.getInjector();
-        ProductService productService = null;
+        ProductService productService = (ProductService) injector.getInstance(ProductService.class);
         List<Product> products = productService.getAllFromFile("products.txt");
```

Comment:
It's good practice to handle exceptions that may be thrown when reading from a file. Consider wrapping this in a try-catch block to handle any IOExceptions that may occur and provide a user-friendly message in case of an error.